### DISCUSSION TITLE
Demo glitch: Long "demo snippets" are cut off

### DISCUSSION THREAD

**Discussion by joewiz** [Jinntec/Fore#198] [Q&A] [answered]
In https://jinntec.github.io/Fore/demo/binding.html, the `<fx-bind>` portion of the demo snippet is cut off at the right side of the div:

> <img width="816" alt="Screenshot 2023-02-11 at 12 06 22" src="https://user-images.githubusercontent.com/59118/218271089-40a6df5f-62bf-49c9-a3a4-a84aaf18fe7b.png">

Using View Source, this and other attributes are shown:

```xml
                    <fx-bind ref="item"
                             required="boolean-from-string(../required)"
                             readonly="boolean-from-string(../readonly)"
                             constraint="boolean-from-string(../constraint)"
                             relevant="boolean-from-string(../relevant)">
                    </fx-bind>
```

Short of using view-source, would it be possible to fix the display of the demo-snippets to make them scrollable or for long lines to wrap?

**Comment by JoernT** [comment]
unfortunately this is a limitation of the polymer component which is used here. That's not easily fixable as the whole content is in shadowDOM. 

We have a better component (jinn-codemirror) now but replacing it everywhere is a pile of work. This is certainly something to consider for a future release.

**Comment by DrRataplan** [comment]
Found it. In the `demo-snippet` js file there's a `position: relative`, which stops the scrolling. I removed it and it does not seem to affect any other demo's noticably.

See PR #203 for the fix

**Comment by JoernT** [comment, marked as answer]
merged that - works fine
